### ISSUE TITLE
feat(select): a11y focus on child options

### ISSUE THREAD

**Issue by tjlav5** [angular/material2#10292]
# Bug:
Focus of child options of an md-select is misrepresented by ChromeVox.

#### What is the expected behavior?
Keyboard navigation to a child option should set focus of ChromeVox to the location of the option in the menu.

#### What is the current behavior?
Keyboard navigation to a child option that overflows past the menu sets the focus of ChromeVox to the overflowed-location.

![screenshot 2018-03-06 at 13 56 48](https://user-images.githubusercontent.com/1452216/37052120-586f371e-2146-11e8-8c98-b0cf3b47a935.png) *Note the orange box*

#### What are the steps to reproduce?
1. Visit: [https://material.angular.io/components/select/overview#creating-groups-of-options](https://material.angular.io/components/select/overview#creating-groups-of-options)
2. Enable ChromeVox (likely reproducible in other OS/Screen-Readers too)
3. Keyboard navigate to Pokemon "Mew" (typing M-e-w)

#### What is the use-case or motivation for changing an existing behavior?
Screen readers should be treated as a first-class citizen with regards to product excellence.

#### Which versions of Angular, Material, OS, TypeScript, browsers are affected?
Angular 5.2.3
Chrome OS (ChromeVox) 64.0.3282.167

#### Anything else we should know?
Likely related to any fix of #1670

**Comment by jelbourn** [comment]
This is working as intended; the select uses `aria-activedescendant` instead of moving browser focus. 

When you say ChromeVox, do you mean the Chrome Extension or the screen-reader in ChromeOS? If the former, it is explicitly not supported because it is not meant to be a real user-centric screen-reader.

**Comment by jelbourn** [comment]
Tracking via #11083
